### PR TITLE
contributing: Update Bio-Formats C++ jobs

### PR DESCRIPTION
**Pull request by ghost**
- Update jobs
- Remove tabs

### CODE REVIEW

File: contributing/ci-bio-formats.txt [hunk 1]
```diff
@@ -228,93 +298,93 @@ breaking jobs are listed under the :jenkinsview:`Breaking` view tab of
...
+                #. Builds the Java generated sources using ``ant clean ome-xml-src``
+                #. Builds the C++ generated sources using ``cmake .. && make gensrc``
+                #. Commits all the generated sources and pushes to
+                   :bf_scc_branch:`develop/breaking/generated_sources` branch
+                #. Computes the diff with
+                   :bf_scc_branch:`develop/latest/generated_sources` and archives
+                   the artifacts
 
-	:jenkinsjob:`BIOFORMATS-5.1-breaking-cpp`
+        :jenkinsjob:`BIOFORMATS-5.1-breaking-cpp`
```
Comment (sbesson) [inline contributing/ci-bio-formats.txt]: This should be BIOFORMATS-CPP-5.1-breaking

File: contributing/conf.py [hunk 2]
```diff
@@ -60,6 +60,7 @@
 .. _openmicroscopy.git: https://github.com/openmicroscopy/openmicroscopy
 .. _bioformats.git: https://github.com/openmicroscopy/bioformats
 .. _ome-documentation.git: https://github.com/openmicroscopy/ome-documentation
+.. _ome-cmake-superbuild.git: https://github.com/openmicroscopy/ome-cmake-superbuild
```
Comment (sbesson) [inline contributing/conf.py]: https://github.com/ome/ome-cmake-superbuild

File: contributing/ci-bio-formats.txt [hunk 3]
```diff
@@ -4,69 +4,102 @@ Bio-Formats jobs
...
+
+        -       * Builds the latest native C++ implementation for Bio-Formats
+                * :term:`BIOFORMATS-CPP-5.1-latest`
+                *
+
+        -       * Builds the latest native C++ implementation for Bio-Formats (Unix superbuild)
+                * :term:`BIOFORMATS-CPP-5.1-latest-superbuild`
+                *
+
+        -       * Builds the latest native C++ implementation for Bio-Formats ("Windows superbuild)
```
Comment (sbesson) [inline contributing/ci-bio-formats.txt]: Remove the opening `"` ?

File: contributing/ci-bio-formats.txt [hunk 4]
```diff
@@ -4,69 +4,102 @@ Bio-Formats jobs
...
+
+        -       * Builds the merge native C++ implementation for Bio-Formats (MacOS X homebrew build)
+                * :term:`BIOFORMATS-CPP-5.1-merge-homebrew`
+                *
+
+        -       * Builds the merge native C++ implementation for Bio-Formats (Unix superbuild)
+                * :term:`BIOFORMATS-CPP-5.1-merge-superbuild`
+                *
+
+        -       * Builds the merge native C++ implementation for Bio-Formats ("Windows superbuild)
```
Comment (sbesson) [inline contributing/ci-bio-formats.txt]: Remove the opening `"` ?

File: contributing/ci-release.txt [hunk 5]
```diff
@@ -38,12 +38,24 @@ jobs should be listed under the :jenkinsview:`Release` view.
...
+    -   * Builds the release native C++ implementation for Bio-Formats
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release`
+    -   * Builds the release native C++ implementation for Bio-Formats (Unix superbuild)
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-superbuild`
+    -   * Builds the release native C++ implementation for Bio-Formats ("Windows superbuild)
+        *
+        * :jenkinsjob:`BIOFORMATS-CPP-5.1-release-win-superbuild`
+    -   * Trigger the Bio-Formats C++ release jobs
```
Comment (sbesson) [inline contributing/ci-release.txt]: Minor comment, I would move the position of this line to the top of the Bio-Formats C++ rows in the jobs table to mirror what we currently have for OMERO and Bio-Formats.

**Comment by sbesson** [comment]
A few comments. Once addressed, this should be ready to merge.

**Comment by rleigh-dundee** [comment]
Thanks for picking up the mistakes; these should all be fixed now.

**Comment by sbesson** [comment]
https://ci.openmicroscopy.org/job/CONTRIBUTING-merge-docs/591/ is green. :+1:

**Pull request merged**